### PR TITLE
Adding notif click fixes

### PR DESCRIPTION
**Pull request by vagnihotri**

### CODE REVIEW

File: pushtemplates/src/main/java/com/clevertap/pushtemplates/PTNotificationIntentService.java [hunk 1]
```diff
@@ -71,6 +71,7 @@ private void handleActionButtonClick(Bundle extras) {
                 }
 
             }
+            Utils.raiseNotificationClicked(getApplicationContext(), extras);
```
Comment (darshanclevertap) [inline pushtemplates/src/main/java/com/clevertap/pushtemplates/PTNotificationIntentService.java]: @vagnihotri We don't need this here, `PTNotificationIntentService` will eventually open the app with the DL info and notification clicked will be raised by the `ActivityLifecycleCallbacks` of the app

File: pushtemplates/src/main/java/com/clevertap/pushtemplates/PushTemplateReceiver.java [hunk 2]
```diff
@@ -415,7 +415,7 @@ private void handleRatingNotification(Context context, Bundle extras) {
             int notificationId = extras.getInt(Constants.PT_NOTIF_ID);
             if (extras.getBoolean("default_dl", false)) {
                 notificationManager.cancel(notificationId);
-
+                Utils.raiseNotificationClicked(context,extras);
```
Comment (darshanclevertap) [inline pushtemplates/src/main/java/com/clevertap/pushtemplates/PushTemplateReceiver.java]: @vagnihotri We could add this here or maybe get this launch intent to call `PTNotificationIntentService` What say?

File: pushtemplates/src/main/java/com/clevertap/pushtemplates/PushTemplateReceiver.java [hunk 3]
```diff
@@ -592,6 +592,7 @@ private void handleProductDisplayNotification(Context context, Bundle extras) {
             int notificationId = extras.getInt(Constants.PT_NOTIF_ID);
             if (buynow == extras.getBoolean("buynow", false)) {
                 notificationManager.cancel(notificationId);
+                Utils.raiseNotificationClicked(context,extras);
```
Comment (darshanclevertap) [inline pushtemplates/src/main/java/com/clevertap/pushtemplates/PushTemplateReceiver.java]: @vagnihotri Same question as above. Think about it. I'm not sure

**Review by darshanclevertap — COMMENTED** [review]
@vagnihotri Read the comments.

**Comment by vagnihotri** [comment]
Using PTNotificationIntentService (when that class is present)

**Pull request merged**